src/main/java/resources/CandidateResource.java

```diff
@@ -0,0 +1,58 @@
+package resources;
+
+import com.google.inject.Inject;
+import dao.CandidateDao;
+import domain.Candidate;
+import io.dropwizard.hibernate.UnitOfWork;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+@Path("/candidate")
+@Produces(MediaType.APPLICATION_JSON)
+public class CandidateResource {
+
+    private CandidateDao candidateDao;
+
+    @Inject
+    public CandidateResource(CandidateDao candidateDao){
+        this.candidateDao = candidateDao;
+    }
+
+    @PUT
+    @Path("/create")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @UnitOfWork
+    public Candidate createCandidate(Candidate candidate){
+        candidateDao.create(candidate);
+        return candidate;
+    }
+
+    @GET
+    @Path("/all")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @UnitOfWork
+    public List<Candidate> getAll(){
+        return candidateDao.findAll();
+    }
+
+    @PUT
+    @Path("/update")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @UnitOfWork
+    public Candidate update(Candidate candidate){
+        candidateDao.update(candidate);
+        return candidate;
+    }
+
+    @GET
```

Comment:
`GET` operations should not modify data -- please use `PUT` or `DELETE` instead.